### DISCUSSION TITLE
navigation bar disappears

### DISCUSSION THREAD

**Discussion by include** [squidfunk/mkdocs-material#2419] [Q&A]
Hi,

I am having a weird behavior - each time I click a link in the body of the page; my left navigation bar disappears.
I don't know if this might be a bug or just some misconfiguration I've done.

Any tip? :)

Cheers,
include

**Comment by squidfunk** [comment]
Hard to say without looking at your configuration. Does this also happen with a mint installation or on the official Material for MkDocs docs?
